config/jobs/kubernetes-sigs/kubespray/kubespray-fejta-bot-periodics.yaml

```diff
@@ -0,0 +1,106 @@
+# this file should contain all periodic jobs that use the fejta-bot token
+periodics:
+- name: periodic-test-infra-close
+  interval: 1h
+  decorate: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/commenter:v20190408-5796ba99f
+      command:
+      - /app/robots/commenter/app.binary
+      args:
+      - |-
+        --query=repo:kubernetes-sigs/kubespray
```

Comment:
Alternatively you could just add repo:kubernetes-sigs/kubespray to the existing jobs.

Reply:
I'm not sure of the logic operator syntax in `--query`. Something like `--query="org:kubernetes or repo:kubernetes-sigs/kubespray"` maybe ?

Reply:
repo and org fields are auto-ORed

see https://github.com/search?q=org%3Akubernetes+repo%3Akubernetes-sigs%2Fkubespray

Don't have a preference. Benefit of putting this in kubespray is you get your own OWNERS file and can change it at your leisure.

This upside is also the downside: you have to remember to update/change the job and do so correctly

Reply:
I think we prefer to go this way and we might re-evaluate if fejta-bot gets enabled on the kubernetes-sigs org at some point like discussed in #11054